server/db/create_db.js

```
@@ -42,6 +42,10 @@ function createDatabase(cb) {
     ') ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_unicode_ci;';
 
   pool.getConnection(function(err, conn) {
+    if (err) {
+      log.warn('unable to connect to database');
+      return cb(err);
+    }
```

Fail fast and hard if we can't get a DB connection.